compiler/rustc_const_eval/src/check_consts/check.rs

```diff
@@ -718,17 +727,19 @@ impl<'tcx> Visitor<'tcx> for Checker<'_, 'tcx> {
                         Some(ConstStability {
                             feature: Some(feature),
                             level: StabilityLevel::Unstable { .. },
-                            const_stable_indirect,
                             ..
                         }) => {
                             self.check_op(ops::IntrinsicUnstable {
                                 name: intrinsic.name,
                                 feature,
-                                const_stable_indirect,
+                                const_stable: is_const_stable,
                             });
                         }
                         Some(ConstStability { level: StabilityLevel::Stable { .. }, .. }) => {
-                            // All good.
+                            // All good. Note that a `#[rustc_const_stable]` intrinsic (meaning it
```

Comment:
When would an intrinsic be marked as `rustc_const_stable` but not `rustc_const_stable_intrinsic`? I cannot seem to find any example in this PR.

Reply:
`transmute` is such an intrinsic. It seems silly to require both attributes.